sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1717,6 +1716,11 @@ export interface ModifiedAccessConditions {
 // @public
 export function newPipeline(credential: StorageSharedKeyCredential | AnonymousCredential | TokenCredential, pipelineOptions?: StoragePipelineOptions): Pipeline;
 
+// @public (undocumented)
+export interface OperationTracingOptions {
```

Comment:
Is this a common pattern across our libraries?
A quick search showed that this interface only exist in code-auth and the storage libraries

cc @xirzec

Reply:
It might be that Storage isn't using OperationOptions.

Reply:
Yep, this is why. Not all of OperationOptions was supported by the time we GA'd, so interfaces were copied. But... core-http has OperationTracingOptions so maybe I'm missing the concern.

Reply:
I think the oddity is that OperationTracingOptions isn't exported by core-http... seems like we should export that as it's part of OperationOptions which *is* exported